funding/manager.go

```diff
@@ -4600,10 +4605,26 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 		MinConfs:          msg.MinConfs,
 		CommitType:        commitType,
 		ChanFunder:        msg.ChanFunder,
-		ZeroConf:          zeroConf,
-		OptionScidAlias:   scid,
-		ScidAliasFeature:  scidFeatureVal,
-		Memo:              msg.Memo,
+		// Unconfirmed Utxos which are marked by the sweeper subsystem
+		// are excluded from the coin selection because they are not
+		// final and can be RBFed by the sweeper subsystem.
+		AllowUtxoForFunding: func(u lnwallet.Utxo) bool {
+			// Utxos with at least 1 confirmation are safe to use
+			// for channel openings because they don't bare the risk
+			// of being replaced (BIP 125 RBF).
+			if u.Confirmations > 0 {
+				return true
+			}
+
+			// Query the sweeper storage to make sure we don't use
+			// an unconfirmed utxo still in use by the sweeper
+			// subsystem.
+			return !f.cfg.IsSweeperOutpoint(u.OutPoint)
```

Comment:
Does this mean that we are willing to use 0-conf outputs as long as the sweeper didn't create them? That still seems suspicious, no?

Reply:
Yes that's the current idea. Currently we are only concerned about unconfirmed outputs when they originated from the sweeper subsystem. At least that cost most of the problems for for LSPs using zeroconf outputs when opening a channel. One of the main reasons was that the Anchor Output which was swept after the Commitment Transaction was confirmed (the Funds Recover case not the CPFP case) was also recovered back to the wallet but because it's only worth to spend them at 1sat/vbyte this sweep eventually got replaced by a third party anchor sweep

Reply:
> is supposed to resolve to?

Anchors resolve as well, but if they are not resolved after 16 blocks, they become anyone can spend outputs so they then can be swept by a third party.

> Shouldn't this comment be this? I think you are not targeting pending sweeps as well?

I rather keep the "in use" because I wanna underline that the sweeper might potentially RBF it again whereas just "published" could not make it not that clear why we cannot use it.